incubation-process.md

```diff
@@ -87,19 +87,15 @@ Projects at the Adopted stage fit into the overall architecture of the OpenRail
 
 ### Criteria
 
-*The criteria for stage 3 are still a draft*
-
-*Examples from CNCF incubation process, to be adopted*
-
 * Document that it is being used successfully in production by at least three independent adopters which, in the TC’s judgement, are of adequate quality and scope.
+* Have a public list of project adopters for at least the primary repo (preferably documented in the project's publiccode.yml file). For a specification, have a list of adopters for the implementation(s) of the spec.
 * Have committers from at least two organizations.
```

Comment:
What means "organization"? Independent organizations? Are the contributions significant? Are the contributing organizations significant? Define some rough rules and let the decision to the judgement of the TC.

Diverse contributors provide stability because the project is not depending on the decisions of only one organization. Open governance is the base, but it still could be run by only one organization.

Contributions from diverse companies are important on stage 3.